src/components/SalesSearchResults/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Box, Typography } from "@mui/material";
+import { SalesSearchResult } from "@/types";
+
+import SalesCard from "./components/SalesCard";
+
+type Props = {
+  results?: SalesSearchResult;
+  loading: boolean;
+  error: null | string;
+};
+
+const SalesSearchResults = ({
+  results,
+  loading,
+  error,
+}: Props): JSX.Element => {
+  if (loading) {
+    return <Typography>Loading...</Typography>;
+  }
```

Comment:
empty space pls (consistency)

Reply:
applied in commit: https://github.com/Sergei29/secret-escapes-tech-test/pull/3/commits/ba87d5d994e87cf0e145d855a11fdf857540ed59